content/inferno/time-series/time-series.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Time Series
+author: clone95
+description: This guide aims to show you the Data Science applications you can build for dealing with time series data.
+---
+
+# Index
+
+- [Time Is Money](#Time-Is-Money)
```

Comment:
Have you seen the classifications [here](https://github.com/neomatrix369/awesome-ai-ml-dl/tree/master/time-series_anomaly-detection#time-series--anomaly-detection) -- would be good to align with some ideas from there as well

Reply:
Yes I've considered it thanks

Reply:
I will add the link in the guide